_sections/videocalling-android-studio.md

```diff
@@ -62,9 +62,14 @@ There are four main controls available whilst on the call. You can choose to mut
 </table>
 
 
+
 #### CodeShare
 
-Selecting the share code option starts a collaboration session in your IDE. This means that any file you open will also open in the IDE of the other participant on the call, allowing you to edit together as you would in a google doc. You can even codeshare on files that aren't under source control (such as files ignored in Git) or files from outside the directory (such as external libraries’ source files). 
+Selecting the share code option opens a dialog, which shows the different projects you have open and the different methods of sharing you have available. You can choose to use JetBrains Code With Me to collaborate, or you can use GitLive's proprietary code sharing solution. You can find more information about using Code With Me here. 
```

Comment:
Selecting the share code option opens up a modal, which shows the different projects you have open and the different methods of pair programming that are available. 

eleaborate on when you can share through which method